cmd/reconciler/ip_test.go

```diff
@@ -252,6 +252,48 @@ var _ = Describe("Whereabouts IP reconciler", func() {
 			Expect(clusterWideIPAllocations.Items).To(HaveLen(expectedClusterWideIPs))
 		})
 	})
+
+	Context("a pod in pending state, without an IP in its network-status", func() {
+		const poolName = "pool1"
+
+		var pod *v1.Pod
+		var pool *v1alpha1.IPPool
+
+		BeforeEach(func() {
+			var err error
+			pod, err = k8sClientSet.CoreV1().Pods(namespace).Create(
+				context.TODO(),
+				generatePendingPod(namespace, podName),
+				metav1.CreateOptions{})
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		BeforeEach(func() {
+			pool = generateIPPoolSpec(ipRange, namespace, poolName, pod.Name)
+			Expect(k8sClient.Create(context.Background(), pool)).NotTo(HaveOccurred())
+			Expect(pool.Spec.Allocations).To(ConsistOf(v1alpha1.IPAllocation{
+				PodRef: fmt.Sprintf("%s/%s", namespace, podName),
+			}))
```

Comment:
I probably should just delete this part. This is testing the testing function ...

Reply:
Done.